fix: Add Reviews history page to review teams group page. (fixes #9190)

kivinen opened the pull request:
This implements the Reviews history page for the review teams.

kivinen committed:
Add Reviews history page to review teams group page.

rjsparks commented:
On the first test run, there is only one real failure (mypy). The second failure at
```
 ======================================================================
FAIL: test_session_attendance (ietf.meeting.tests_views.ProceedingsTests.test_session_attendance)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/__w/datatracker/datatracker/ietf/meeting/tests_views.py", line 9323, in test_session_attendance
    self.assertContains(r, person.plain_name())
  File "/usr/local/lib/python3.9/site-packages/django/test/testcases.py", line 660, in assertContains
    self.assertTrue(
AssertionError: False is not true : Couldn't find 'David O'Connor' in response
```
is a transient.
The rest are just consequences of the first two.

For the first error, try 
```
HistoricalReviewAssignment = ReviewAssignment.history.model
```
just before using it in the code rather than importing HistoricalReviewAssignment.

kivinen committed:
Merge branch 'ietf-tools:main' into fix-9190

kivinen committed:
fix: Added test cases.

Fixed html to validate, moved sort text to data-text from comments.
Added test cases. fixes #9190
